pkg/placement/membership.go

```diff
@@ -366,19 +372,29 @@ func (p *Service) performTablesUpdate(ctx context.Context, hosts []placementGRPC
 		}
 	}
 
+	// Enforce maximum API level
```

Comment:
Perhaps this enforcement can be done just once for both `newTable` and `newTableWithVirtualNodes`, where we compute the "final" API level

Reply:
We could have any combination of  `newTable` and `newTableWithVirtualNodes` being present, so I preferred leaving them separate like this, so we don't have to "untangle" them in a few months when we have to remove `newTableWithVirtualNodes`.

Reply:
Ok